pyproject.toml

```diff
@@ -93,10 +97,11 @@ build-backend = "poetry.core.masonry.api"
 [tool.pytest.ini_options]
 addopts = ["--import-mode=importlib"]
 # see https://docs.pytest.org/en/latest/explanation/goodpractices.html#tests-outside-application-code
+pythonpath = [".", "src", "test"]
```

Comment:
the only real change in this file, which was needed for the tests to run (because I'm importing stuff from within the test package - i.e. `tests/unittests/helpersconnection_mock.py`), all other changes are auto-formatting... sorry about that